src/workloads/scale/CreateDropView.yml

```diff
@@ -0,0 +1,81 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload loads a collection with documents and then repeatedly
+  create and drops a view on that collection.
+Keywords:
+- view
+- create
+- drop
+
+dbname: &dbname viewDB
+DocumentCount: &NumDocs 100000
+CollectionCount: &NumColls 1
+
+
+Actors:
+- Name: Setup
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    BatchSize: 100
+    Threads: 1
+    DocumentCount: *NumDocs
+    Database: *dbname
+    CollectionCount: *NumColls
+    Document: &doc
+      id: {^RandomInt: {min: 0, max: *NumDocs}}
+      a: {^RandomInt: {min: 0, max: *NumDocs}}
+      c: &string {^RandomString: {length: 50}}  # Adjust this so the doc comes out as 100 B.
+    Indexes:
+    - keys: {id: 1}
+    - keys: {a: 1}
+  - Phase: 1..3
+    Nop: true
+
+- Name: QuiesceBetweenLevels
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - &nop {Nop: true}
+  - &quiesce
+    Repeat: 1
+    Database: admin
+    Operations:
+    # Fsync to force a checkpoint and quiesce the system.
+    - OperationMetricsName: FsyncCommand
+      OperationName: AdminCommand
+      OperationCommand:
+        fsync: 1
+  - *nop
+  - *quiesce
+
+# This actor repeatedly creates and then drops a view
+- Name: ViewCreateDrop
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - &toggleView1
+    Repeat: 100000
+    Database: *dbname
+    Operations:
+    - OperationMetricsName: CreateViewMetric
+      OperationIsQuiet: true
+      OperationName: RunCommand
+      OperationCommand:
+        {create: myView, viewOn: Collection0, pipeline: [{$match: {id: {$lt: 10}}}]}
+    - OperationName: RunCommand
+      OperationMetricsName: DropViewMetric
+      OperationIsQuiet: true
+      OperationCommand:
+        drop: myView
+  - Nop: true
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - single-replica
```

Comment:
Is it interesting to run in other configurations? I'm wondering if there's any interactions with replication or sharding. Thoughts. @jimoleary ?

Reply:
maybe add replica and shard-lite?

Reply:
Sure thing - I added replica, and shard-lite. I also removed single-replica as it seemed unnecessary if we're including a run on a full replica set.